src/commands/lambda/flare.ts

```diff
@@ -218,31 +470,18 @@ export const sendToDatadog = async (
 
   try {
     await axios.post(ENDPOINT_URL, form, headerConfig)
-    context.stdout.write('\n✅ Successfully sent flare file to Datadog Support!\n')
   } catch (err) {
+    // Ensure the root folder is deleted if the request fails
+    deleteFolder(rootFolderPath)
+
     const errResponse: string = err.response?.data?.error
     throw Error(`Failed to send flare file to Datadog Support: ${err.message}. ${errResponse ?? ''}\n`)
```

Comment:
Note that `err.response` does not exist when `err instanceof Error`, so I did not type guard here